VultisigApp/VultisigApp/Views/Keygen/JoinKeygenView.swift

```diff
@@ -170,22 +173,20 @@ struct JoinKeygenView: View {
         .cornerRadius(12)
     }
     
+    var shadow: some View {
+        Circle()
+            .frame(width: 360, height: 360)
+            .foregroundColor(.alertTurquoise)
+            .opacity(0.05)
+            .blur(radius: 20)
+    }
+    
     var joinKeygen: some View {
-        VStack {
-            HStack {
-                Text("thisDevice")
-                Text(self.viewModel.localPartyID)
-            }
-            
-            HStack {
-                Text(NSLocalizedString("joinKeygen", comment: "Joining key generation, please wait..."))
-                    .onAppear {
-                        viewModel.joinKeygenCommittee()
```

Comment:
@Rockindash `viewModel.joinKeygenCommittee()` this one can't be removed ,  this line is the device actually join the comittee